contrib/for-tests/volumes/gluster/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:14.04
```

Comment:
As per previous comment, can we not get by with a smaller base container image here.  This one includes "ubuntu universe" which seems excessive.